web/templates/settings/upload_space_stats.hbs

```diff
@@ -1,9 +1,11 @@
 <span>
-    {{t "Organization using {percent_used}% of {upload_quota}." }}
     {{#if show_upgrade_message}}
-        {{#tr}}
-            <z-link>Upgrade</z-link> for more space.
-            {{#*inline "z-link"}}<a href="/upgrade/" target="_blank" rel="noopener noreferrer">{{> @partial-block }}</a>{{/inline}}
-        {{/tr}}
+    <a href="/upgrade/" class="upgrade-tip" target="_blank" rel="noopener noreferrer">
+        {{upload_quota_string}} {{t "Upgrade for more space." }}
+    </a>
+    {{else}}
+    <div class="upgrade-tip">
```

Comment:
Think we can avoid using `upgrade-tip` class when not mentioning about upgrading.

Reply:
So @sahil839, from your pov, should I use "tip" here or define a new class with the same properties as "upgrade-tip"?

Reply:
You can use `tip` here.